frontend/encore/faq.rst

```diff
@@ -137,3 +137,30 @@ Babel. But, you can change that via the ``configureBabel()`` method. See
 :doc:`/frontend/encore/babel` for details.
 
 .. _`rsync`: https://rsync.samba.org/
+
+How Do I Integrate my Webpack Configuration in my IDE?
+------------------------------------------------------
+
+Some IDE (like `PhpStorm <https://www.jetbrains.com/help/phpstorm/using-webpack.html>`_) can read and interpret your ``webpack.config.js`` file
+to make your development easier (e.g.: aliases resolution).
+
+However, you will face the following error:
```

Comment:
I'd put `could` instead of `will` there since the check may end-up in the flex recipe.